tests/shared/src/test/scala/org/typelevel/ci/CIStringSuite.scala

```diff
@@ -123,6 +123,51 @@ class CIStringSuite extends DisciplineSuite {
     CIString("  text   ").trim == CIString("text")
   }
 
+  property("ci interpolator is consistent with apply") {
+    forAll { (s: String) =>
+      assertEquals(ci"${s}", CIString(s))
+    }
+  }
+
+  property("ci interpolator handles expressions") {
+    forAll { (x: Int, y: Int) =>
+      assertEquals(ci"${x + y}", CIString((x + y).toString))
+    }
+  }
+
+  property("ci interpolator handles multiple parts") {
+    forAll { (a: String, b: String, c: String) =>
+      assertEquals(ci"$a:$b:$c", CIString(s"$a:$b:$c"))
+    }
+  }
+
+  property("ci interpolator extractor is case-insensitive") {
+    forAll { (s: String) =>
+      CIString(new String(s.toString.toArray.map(_.toUpper))) match {
```

Comment:
I think this needs to be wrapped in `assert`.

Reply:
Returning a bool is also fine, so only the assert here was strictly necessary, but I guess it is nice for consistency to add it everywhere.